xl_auth/user/views.py

```diff
@@ -112,12 +112,21 @@ def inspect(user_id):
         return redirect(get_redirect_target())
     else:
         tokens = Token.query.filter_by(user=user).all()
+
         permissions_created = Permission.query.filter_by(created_by=user).count()
```

Comment:
We probably should rename these two fields now to something like `num_permissions_{created,modified}`..

Reply:
- [ ] prefix it